interfaces/matlab_experimental/Base/Solution.m

```diff
@@ -39,57 +31,62 @@
     %
     % :param src:
     %     Input string of YAML file name.
-    % :param id:
-    %     ID of the phase to import as specified in the YAML file.
-    % :param trans:
-    %     String, transport modeling. Possible values are ``'default'``, ``'none'``,
-    %     ``'mixture-averaged'``, or ``'multicomponent'``. If not specified,
-    %     ``'default'`` is used.
+    % :param name:
+    %     name of the phase to import as specified in the YAML file.
+    % :param transport_model:
+    %     String specifying transport model. Possible values are ``'default'``,
+    %     ``'none'``, ``'mixture-averaged'``, or ``'multicomponent'``. If not specified,
+    %     ``'none'`` is used.
```

Comment:
For consistency with the Python and C++ interfaces, shouldn't this use the `default` transport model if the argument is omitted?

Reply:
Whoops - good catch!